docs/_basic/ja_ listening_responding_shortcuts.md

```diff
@@ -0,0 +1,65 @@
+---
```

Comment:
@seratch this file will need to be updated based on changes in https://github.com/slackapi/bolt/pull/428/files#diff-a99cc45487ce0e4ce5abb83cfdd04150